recipes/rttr/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+import os
+import glob
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+
+class RTTRConan(ConanFile):
+    name = "rttr"
+    description = "Run Time Type Reflection library"   
+    topics = ("conan", "reflection", "rttr", )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rttrorg/rttr"
+    license = "MIT"
+
+    exports_sources = "CMakeLists.txt", "patches/*.patch",
+    generators = "cmake",
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_rtti": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_rtti": False,
+    }
+
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows" or self.options.shared:
```

Comment:
fPIC flag does not apply when building in static, so there is no need to add this check
```suggestion
        if self.settings.os == "Windows":
```